aten/src/ATen/native/native_functions.yaml

```diff
@@ -14120,7 +14120,7 @@
     CUDA: _efficient_attention_forward
   tags: nondeterministic_seeded
 
-- func: _efficient_attention_backward(Tensor grad_out_, Tensor query, Tensor key, Tensor value, Tensor? bias, Tensor out, Tensor? cu_seqlens_q, Tensor? cu_seqlens_k, int max_seqlen_k, int max_seqlen_q, Tensor logsumexp, float dropout_p, Tensor rng_seed, Tensor rng_offset, int custom_mask_type, *, float? scale=None) -> (Tensor, Tensor, Tensor, Tensor)
+- func: _efficient_attention_backward(Tensor grad_out_, Tensor query, Tensor key, Tensor value, Tensor? bias, Tensor out, Tensor? cu_seqlens_q, Tensor? cu_seqlens_k, int max_seqlen_k, int max_seqlen_q, Tensor logsumexp, float dropout_p, Tensor rng_seed, Tensor rng_offset, int custom_mask_type, int num_splits_key, *, float? scale=None) -> (Tensor, Tensor, Tensor, Tensor)
```

Comment:
Seems like num_splits_key is more of an optional type than an int? You're using "-1" to say "pick a default".

Reply:
It is another performance lever thatt in theory could be pulled, and is exposed in xformers. Setting it to -1 says "use the heuristic to choose the number of splits.  It is more oprtional field from the persepctive of our API. I can update

Reply:
updated